src/components/Header.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import Text from "./Text";
+import Title from "./Title";
+import background from "../assets/renature/background.svg";
```

Comment:
Let's have this fallback to a gradient based on background color when the user doesn't provide a background image in the config, or when it is not found